_examples/8-questions-to-fit-candidate-into-team.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 8 Questions, to fit candidate into team
+author: Ievgenii Vdovenko
+role:  interviewer
+overview: Aim of this questions is to build happy team. My believe is - if team is happy, it's more productive.
+questions:
+- question: Describe your vision of role you applied to.
+  summary: very often candidate and company think differently about Role, so need sync up such detail; 
+- question: Deal breakers, or what situation/environment will immediately trigger red flag in your head.
+  summary: important information, because may be your env./team/etc. is bad for candidate, and could lead to future problems for you and candidate
+- question: Evaluate yourself in {technology/language/etc.} in scale of 0-10
+  summary: first ask self assigned mark, and then double check what this mean for candidate.   
+- question: Cultural fit - this kinda not question, more action, before making decision need introduce candidate to team
+  summary: doesnt matter how good candidate is, but if he/she can't work within specific team - it will be disaster  
```

Comment:
doesn't matter how good a candidate is, but if he/she can't work within a specific team - it will be a disaster